src/components/Home/WhatsNewModal/index.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React, { useState } from 'react'
+import cn from 'classnames'
+import FocusLock from 'react-focus-lock'
+
+import Video from '../UseCases/Video'
+import Link from '../../Link'
+
+import { ReactComponent as CloseSvg } from '../../../../static/img/close-icon.svg'
+
+import * as styles from './styles.module.css'
+
+const version = '2.8.3'
```

Comment:
I don't think we need version for now. It's only for this specific "release," which is not strongly tied to new features in any DVC version.

Reply:
Makes sense! Would the button saying "What's New" work?

Reply:
Yup, "What's new" works!